### PR TITLE
Export more command functions

### PR DESCRIPTION
**Pull request by gregchanman**
Exposing these functions so I can use in my own commands.

### CODE REVIEW

**Comment by RatoX** [comment]
Hey @gregchanman, I'm not sure if we want to expose those API.

@eshvedai what do you think?

File: index.d.ts [hunk 1]
```diff
@@ -217,6 +239,12 @@ export function addColumnBefore<S extends Schema = any>(
   dispatch?: (tr: Transaction<S>) => void
 ): boolean;
 
+export function addColumn<S extends Schema = any>(
```
Comment (marduke182) [inline index.d.ts]: Need to pass `<S>` to `Transaction<S>`

File: index.d.ts [hunk 2]
```diff
@@ -202,6 +218,12 @@ export function addRowBefore<S extends Schema = any>(
   dispatch?: (tr: Transaction<S>) => void
 ): boolean;
 
+export function addRow<S extends Schema = any>(
+  transaction: Transaction,
```
Comment (marduke182) [inline index.d.ts]: Need to pass `<S>` to `Transaction<S>`

File: index.d.ts [hunk 3]
```diff
@@ -192,6 +198,16 @@ export function deleteRow<S extends Schema = any>(
   dispatch?: (tr: Transaction<S>) => void
 ): boolean;
 
+export function selectedRect<S extends Schema = any>(
+  state: EditorState<S>
+): TableRect;
+
+export function rowIsHeader<S extends Schema = any>(
+  map: TableMap,
+  table: Node,
```
Comment (marduke182) [inline index.d.ts]: Use `ProsemirrorNode<S>`, the default node is `HTMLNode`.

**Comment by gregchanman** [comment]
> Please update documentation with the new exported methods.

What section would you have it under? They're more like util functions (which are not documented) than commands.

**Comment by marduke182** [comment]
@gregchanman mm let me discuss with @eshvedai and @RatoX on Monday about where and how we can export this.

**Comment by gregchanman** [comment]
> @gregchanman mm let me discuss with @eshvedai and @RatoX on Monday about where and how we can export this.

Any update?

**Comment by eshvedai** [comment]
LGTM 👍

**Pull request merged**